action.yml

```diff
@@ -29,11 +29,16 @@ inputs:
     description: Should Cygwin's bin directory be added to the system PATH?
     required: false
     default: true
+  check-hash:
+    description: Check the hash of the installer
+    required: false
+    default: 'true'
 
 runs:
   using: "composite"
   steps:
     - run: |
+        $ErrorActionPreference = 'Stop'
```

Comment:
I guess this modifies the behaviour of Invoke-WebRequest?

Reply:
That's just to be sure that the whole action fails in case we have an error (from `Invoke-WebRequest` or even an explicitly thrown exception).

For example, try these comamnds in a pwsh shell:

```pwsh
$ErrorActionPreference = 'SilentlyContinue'; throw 'Error!'; Write-Host 'After exception'
```

You'll see

```
After exception
```

But with

```pwsh
$ErrorActionPreference = 'Stop'; throw 'Error!'; Write-Host 'After exception'
```

you'll have

```
Exception: Error!
```